FEM/rf_random_walk.cpp

```diff
@@ -794,8 +794,8 @@ void RandomWalk::InterpolateVelocityOfTheParticleByInverseDistance(Particle* A)
 					P[p] = crossroad->plane[k].norm[p];
```

Comment:
actually there is no real change in rf_random_walk.h and .cpp. Can you revert the changes? The code formatting changes should be done in another PR.